### ISSUE TITLE
Dreambooth Swift Diffusion?

### ISSUE THREAD

**Issue by enzyme69** [liuliu/swift-diffusion#11]
Do you have plan to implement Dreambooth for Swift Diffusion? I am super curious on this, been looking around. But usually Windows or Linux?

**Comment by liuliu** [comment]
Yeah, one thing good about my implementation is that s4nnc is a good framework for training as well. It has been used to train from RL models to image classifiers to translation models. However, we do need to implement backward passes for all the ops involved, and need to take a closer look at memory usages (as I understand, many of the current "dreambooth" implementations uses 8bit NVIDIA adam optimizer implemented in bitsandbytes repo).
